### PR TITLE
Test for CASSANDRA-9871

### PR DESCRIPTION
**Pull request by stef1927**
A simple test for CASSANDRA-9871.

### CODE REVIEW

File: bootstrap_test.py [hunk 1]
```diff
@@ -422,3 +422,42 @@ def failed_bootstap_wiped_node_can_join_test(self):
         mark = node2.mark_log()
         node2.start(wait_other_notice=True)
         node2.watch_log_for("JOINING:", from_mark=mark)
+
+    @require("9871")
```
Comment (ptnapoleon) [inline bootstrap_test.py]: No need for this now that it has been committed.

File: bootstrap_test.py [hunk 2]
```diff
@@ -422,3 +422,42 @@ def failed_bootstap_wiped_node_can_join_test(self):
...
+        if version < "2.1":
+            node1.stress(['-n', '10000'])
+        else:
+            node1.stress(['write', 'n=10000', '-rate', 'threads=8'])
+
+        # Stop node 3
+        node3.stop(gently=True)
+
+        # Sleep a bit to let GOSSIP settle
+        time.sleep(2)
```
Comment (ptnapoleon) [inline bootstrap_test.py]: Wouldn't using wait_other_notice=True in the `node3.stop()` call remove the need to let gossip settle?

**Comment by ptnapoleon** [comment]
This test should go in replace_address_test.py, not bootstrap_test.py. Also, any ideas why the existing replace_address tests didn't catch 9871? Was it really just the gently=True stop? That's upsetting.

**Comment by stef1927** [comment]
Moved the test to replace_address_test.py and adapted it to reuse the existing test, which requires indeed gently=True in order to reproduce the problem.

**Comment by ptnapoleon** [comment]
Thanks! +1

**Pull request merged**